demo/auth/passport.js

```diff
@@ -0,0 +1,47 @@
+var $fh = require('fh-js-sdk');
+var logger = require('@raincatcher/logger').logger;
+var ConsoleLogger = require('@raincatcher/logger').ConsoleLogger;
+var setLogger = require('@raincatcher/logger').setLogger;
+
+$fh.init({}, function() {
+  var passport = {
+    loadUserProfile: function($http, $window) {
+      var req = {
+        method: 'GET',
+        url: $fh.getCloudURL() + '/profile'
```

Comment:
Hardcoding urls in logic is antipattern. This integration is part of the demo application but it's still reflecting some decisions made on server. Let's make sure that we have proper abstraction for this.